chalicelib_smaht/checks/wfr_checks.py

```diff
@@ -115,8 +118,8 @@ def md5runsmaht_status(connection, file_type="", start_date=None, **kwargs):
         check.allow_action = False
         return check
```

Comment:
Waiting for the index queue to clear is kind of annoying here, but may not be an issue for now. An alternative is to check the database for the files returned from search to ensure everything is up-to-date, but can add this if the indexing queue wait time becomes a problem.